test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 603000
-//   executionCost: 638
-//   totalCost: 603638
+//   codeDepositCost: 631600
+//   executionCost: 664
+//   totalCost: 632264
```

Comment:
Weird that the gas costs increased by a lot, even though the optimised code looks better (?)

Here is a diff comparing removing-unused-returns + removing-unused-function-parameters v/s only removing-unused-function-arguments
https://gist.github.com/hrkrshnn/7c14cfa1794271792fbf5f36e0d6cfda

@chriseth What do you think lead to the increase?